programs/test/zeroize.c

```diff
@@ -98,5 +96,6 @@ int main( int argc, char** argv )
     fclose( fp );
     mbedtls_platform_zeroize( buf, sizeof( buf ) );
 
+    asm( "exit_label:" );
```

Comment:
Unfortunately, this only works on compilers that support this `asm` syntax. CI says that Visual Studio doesn't.

This program should be as portable as possible, because everyone should run it on their platform with their compiler and use their platform's debugger to check the result. We only provide a debugger script with gdb, but the program itself should be usable on platforms that don't have gdb.

Reply:
ACK, I have switched to a method that only adds a magic comment in the C source, preserving it's portability.